demo/src/screens/componentScreens/DateTimePickerScreen.js

```diff
@@ -1,7 +1,32 @@
 import React, {Component} from 'react';
-import {DateTimePicker, View, Text} from 'react-native-ui-lib'; // eslint-disable-line
+import {DateTimePicker, View, Text, Button} from 'react-native-ui-lib'; // eslint-disable-line
 
 export default class DateTimePickerScreen extends Component {
+
+  getCustomInputValue = value => {
+    if (!value) {
+      return '';
+    }
+    return value.includes(new Date().getFullYear() + 1) ? 'Next Year' : value;
+  };
+
+  renderCustomInput = (props, toggle) => {
+    const {value} = props;
+    return (
+      <View>
+        <Button
+          label={'Select a date'}
+          style={{backgroundColor: 'red', height: 40, width: '100%'}}
+          {...props}
+          onPress={() => {
+            toggle(true);
+          }}
+        />
+        <Text>{this.getCustomInputValue(value)}</Text>
+      </View>
+    );
+  };
```

Comment:
Maybe we can do it a little prettier 😬 
Like in the design of the ticket

Reply:
Lol np